src/adapters/PeopleAdapter.js

```diff
@@ -0,0 +1,13 @@
+import {throwError} from 'rxjs';
+
+/**
+ * This is the base class that `PeopleAdapter` will inherit from.
+ * When creating your subclass `peopleAdapter`, make sure all methods
+ * are defined and the items returned have the proper shape.
+ */
+export default class PeopleAdapter {
+  // eslint-disable-next-line no-unused-vars
+  getPerson(id) {
```

Comment:
Add a docstring for `getPerson`, specifying what it does, parameter descriptions and return values.
When we add the data object, we should add a link here for a sample of the expected return structure.